ui/app/router.js

```diff
@@ -59,9 +59,13 @@ Router.map(function () {
           this.route('section', { path: '/:section_name' });
         });
         this.route('mfa', function () {
-          // lookup
-          this.route('index', { path: '/' });
           this.route('configure', { path: '/landing' });
+          this.route('methods', { path: '/' });
+          this.route('method', { path: '/method/:method_id' }, function () {
+            this.route('index', { path: '/' });
```

Comment:
Is it necessary to define an index route here?

Reply:
I don't think, I just kept as a placeholder for adding future routes.

Reply:
Ok sounds good :+1: